storybooknative/src/InputField/index.js

```diff
@@ -1,30 +1,113 @@
 // @flow strict
 
 import * as React from "react";
-import { TextInput } from "react-native"; // eslint-disable-line no-restricted-imports
+import { TextInput, View } from "react-native"; // eslint-disable-line no-restricted-imports
```

Comment:
I don't think you have restricted imports for this project. You can delete this eslint-disable